Objective-C/Tests/VectorSearchTest.m

```diff
@@ -76,12 +76,11 @@ - (void) setUp {
     AssertNotNil(_wordDB);
     
     _logger = [[CustomLogger alloc] init];
-    _logger.level = kCBLLogLevelInfo;
-    CBLDatabase.log.custom = _logger;
+    CBLLogSinks.custom = [[CBLCustomLogSink alloc] initWithLevel: kCBLLogLevelInfo logSink: _logger];
```

Comment:
I don't remember why we have a custom logger set for the VectorSearchTest?

Reply:
It is used to check for `Untrained index; queries may be slow`. I remember a conversation about this vaguely, but I wasn't sure enough to remove such checks. So I kept the custom logger in place. Do we still want to check for untrained? If not, I will remove this logger altogether.

Reply:
I see. Let's keep it for now I think.